### PR TITLE
Add core and safe versions of change_argp.

### PR DESCRIPTION
**Pull request by Syed-Osama-Hussain**
Addresses [1355](https://github.com/poliastro/poliastro/issues/1355) for change_argp.

### CODE REVIEW

File: src/poliastro/twobody/thrust/change_argp.py [hunk 1]
```diff
@@ -23,24 +19,16 @@ def change_argp(k, a, ecc, argp_0, argp_f, f):
...
-
-    delta_V, t_f = extra_quantities(k, a, ecc, argp_0, argp_f, f, A=0.0)
-
-    return a_d, delta_V, t_f
+    a_d, delta_V, t_f = change_a_inc_fast(
+        k,
+        a,
+        ecc,
+        argp_0,
+        argp_f,
```
[the remark this answers is not in the document]
Comment (astrojuanlu) [reply src/poliastro/twobody/thrust/change_argp.py:30]: - k: Standard Gravitational parameter (km^3/s^2)
- a: Semi-major axis (km)
- ecc: Eccentricity
- argp_0: Initial argument of periapsis (rad)
- argp_f: Final argument of periapsis (rad)

[the remark this answers is not in the document]
Comment (Syed-Osama-Hussain) [reply src/poliastro/twobody/thrust/change_argp.py:30]: Thank you! I'll update it.

File: tests/tests_twobody/test_thrust.py [hunk 2]
```diff
@@ -287,7 +287,7 @@ def f_soyuz(t0, u_, k):
 
     sf = s0.propagate(t_f, method=cowell, f=f_soyuz, rtol=1e-8)
 
-    assert_allclose(sf.argp.to(u.rad).value, argp_f, rtol=1e-4)
+    assert_allclose(sf.argp.to(u.rad).value, (argp_f).value, rtol=1e-4)
```
Comment (astrojuanlu) [inline tests/tests_twobody/test_thrust.py:290]:
```suggestion
    assert_allclose(sf.argp.to_value(u.rad), argp_f.to_value(u.rad), rtol=1e-4)
```

**Review by astrojuanlu — APPROVED** [review]
Looks great, thanks @Syed-Osama-Hussain !

**Pull request merged**